src/components/navigation/search-input.tsx

```diff
@@ -65,6 +65,32 @@ export function SearchInput({ onSearchPage }: SearchInputProps) {
       input.setAttribute('placeholder', strings.header.search)
       input.focus()
       setSearchActive(true)
+
+      const script = document.createElement('script')
+      script.innerHTML = `
+    var resultsContainer = document.getElementById('gcs-results');
+    setupLinkCatcher(resultsContainer);
+
+    function setupLinkCatcher(container){
+      if(!container || container === undefined) return;
+      var className = 'gs-title';
+      
+      document.addEventListener('click', function (e) {
```

Comment:
can we add the click-hanlder only to the container?

Reply:
Thanks… forgot to change that back.